src/components/backend-ai-settings-view.ts

```diff
@@ -50,16 +50,25 @@ export default class BackendAiSettingsView extends BackendAIPage {
   ];
   @property({type: Array}) jobschedulerType = [
     'fifo', 'lifo', 'drf'];
+  @property({type: String}) scheduler = 'fifo';
 
   constructor() {
     super();
+    try {
+      this.scheduler = localStorage.getItem('backendaiclient.settings.scheduler') || 'fifo';
```

Comment:
How about using [backend-ai-settings-store](https://github.com/lablup/backend.ai-webui/blob/main/src/components/backend-ai-settings-store.ts) component instead of using localStorage directly?